src/test/util_tests.cpp

```diff
@@ -50,19 +50,19 @@ BOOST_AUTO_TEST_CASE(util_datadir)
     ClearDatadirCache();
     const fs::path dd_norm = GetDataDir();
 
-    gArgs.ForceSetArg("-datadir", dd_norm.string() + "/");
+    gArgs.ForceSetArg("-datadir", dd_norm.string());
```

Comment:
Why this change? Does it break with `+ "/"`?

Reply:
> Why this change? Does it break with + "/"?

Yes, at ef8c98b1a54280f279f5319f386c44026893bf1f this currently fails with the trailing slash:
```bash
error: in "util_tests/util_datadir": check dd_norm == GetDataDir() has failed ["/var/folders/z2/cn877pxd3czdfh47mfkmbwgm0000gn/T/test_common_Bitcoin Core/873b1e644e770a9d5cdfc8f89323a344256001e7cedc14e18f2ae7c4a8ea8f94" != "/var/folders/z2/cn877pxd3czdfh47mfkmbwgm0000gn/T/test_common_Bitcoin Core/873b1e644e770a9d5cdfc8f89323a344256001e7cedc14e18f2ae7c4a8ea8f94/"]
```

Reply:
I've added to the commit message, so will mark this as resolved.